docs/fidesops/docs/guides/database_connectors.md

```diff
@@ -131,15 +131,15 @@ PUT api/v1/connection/my-mongo-db/secret?verify=false`
 
 ### Testing your connection 
 
-You can verify that a ConnectionConfig's secrets are valid at any time by calling the [Test a ConnectionConfig's Secrets](/api#operations-Connections-test_connection_config_secrets_api_v1_connection__connection_key__test_get) operation:
+You can verify that a ConnectionConfig's secrets are valid at any time by calling the [Test a ConnectionConfig's Secrets](/fidesops/api#operations-Connections-test_connection_config_secrets_api_v1_connection__connection_key__test_get) operation:
 
 
 ```
 GET /api/v1/connection/application-postgresql-db/test
 ```
 
 Once again, the `test_status` and `failure_reason` properties describe the success or failure of the test. If the test failed,
-you should adjust the ConnectionConfig Secrets properties through additional calls to [Set a ConnectionConfig's Secrets](/api#operations-ConnectionConfigs-put_connection_config_secrets_api_v1_connection__connection_key__secret_put)
+you should adjust the ConnectionConfig Secrets properties through additional calls to [Set a ConnectionConfig's Secrets](/fidesops/api#operations-ConnectionConfigs-put_connection_config_secrets_api_v1_connection__connection_key__secret_put)
```

Comment:
internal portion of this link is broken too - 
`/fidesops/api#operations-Connections-put_connection_config_secrets_api_v1_connection__connection_key__secret_put`

Reply:
these are weird, i diffed with the appropriate path and it seems to be working, which has no diff. So I dirtied and readded, i pray this is the last time!